cmd/preset.go

```diff
@@ -124,3 +194,10 @@ func NewPresetCommand(preset *KoolPreset) (presetCmd *cobra.Command) {
 	presetCmd.Flags().BoolVarP(&preset.Flags.Override, "override", "", false, "Force replace local existing files with the preset files")
 	return
 }
+
+func formatTemplateKey(key string) (formattedKey string) {
```

Comment:
took me a while to track the code path till here... I can see how it works, but we do need to improve/refactor. This for example should not be necessary if we have better structured  presets data to start with.

Reply:
This is too indirect/not-intuitive.